ghcide/src/Development/IDE/Types/Exports.hs

```diff
@@ -121,7 +146,8 @@ createExportsMapHieDb hiedb = do
         let mn = modInfoName $ hieModInfo m
             mText = pack $ moduleNameString mn
         fmap (wrap . unwrap mText) <$> getExportsForModule hiedb mn
-    return $ ExportsMap $ Map.fromListWith (<>) (concat idents)
+    let exportsMap = Map.fromListWith (<>) (concat idents)
+    return $ ExportsMap exportsMap Map.empty
```

Comment:
Why is the module exports map empty in this case? It shouldn't be, we've got all the information needed to construct one

Reply:
addressed